### PR TITLE
[DOCFIX] Update partitioned table instructions

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
@ZacBlanco PTAL

**Comment by yuzhu** [comment]
jenkins, test this please

File: docs/en/compute/Hive.md [hunk 1]
```diff
@@ -195,6 +195,16 @@ Instructions and examples till here illustrate how to use Alluxio as one of the
 tables in Hive, together with other filesystems like HDFS. They do not require to change the global
 setting in Hive such as the default filesystem which is covered in the next section.
 
+### Move an Partitioned Table
```
Comment (gpang) [inline docs/en/compute/Hive.md]:
```suggestion
### Move a Partitioned Table
```

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by gpang** [comment]
alluxio-bot, cherry-pick to branch-2.2 please

**Comment by gpang** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please

**Pull request merged**